### ISSUE TITLE
[Feature Request]: Task Entry Behavior Modification

### ISSUE THREAD

**Issue by rsnj5** [vigneshs-dev/golang-to-do-app#5]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

Description:

Currently, when users enter a new task, the tick button is already marked as completed (ticked). This feature request aims to change this behavior so that the tick button is left blank for new tasks. Users should manually tick the button upon completing the task.

Expected Behavior:

When a new task is created, the tick button should be in an unchecked state.
The tick button should only be checked once the user manually marks the task as completed.

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I want to work on this issue
- [X] I'm willing to provide further clarification or assistance if needed.

**Comment by rsnj5** [comment]
I can work on this issue

**Comment by vigneshs-dev** [comment]
Sure @rsnj5 . Your point is valid. You can start working on this issue. Keep upstream from the main branch before starting. Thankyou for your contribution.
